engine/mcpinstall.go

```
@@ -0,0 +1,81 @@
+	if err != nil {
+		return errors.New("cannot determine user home directory")
+	}
+
+	agentDir := filepath.Join(home, meta.dir)
+	if err := os.MkdirAll(agentDir, 0755); err != nil {
+		return fmt.Errorf("create %s: %w", agentDir, err)
+	}
+
+	configPath := filepath.Join(agentDir, meta.file)
+	var cfg mcpConfig
+	if data, err := os.ReadFile(configPath); err == nil {
```

golangci-lint (2.13.2)

[medium] G304: Potential file inclusion via variable

(gosec)